tools/Azure.Mcp.Tools.Deploy/src/Commands/Architecture/DiagramGenerateCommand.cs

```diff
@@ -69,6 +69,11 @@ public override Task<CommandResponse> ExecuteAsync(CommandContext context, Parse
                 throw new ArgumentException($"Invalid JSON format: {ex.Message}", nameof(rawMcpToolInput), ex);
             }
 
+            context.Activity?
```

Comment:
* Can these tags be consolidated into a constants file?
* Since the repo has many different areas in it, there would be less chance of collision using `{area}/{tag_name}"... (i.e. deploy/ServiceCount)